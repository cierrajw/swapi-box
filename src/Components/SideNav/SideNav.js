import React from 'react';

const SideNav = () => {

  return (
    <div>
    The SideNav
    </div>
  );
}

export default SideNav;
