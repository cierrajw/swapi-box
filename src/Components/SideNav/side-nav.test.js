import React from 'react';
import { shallow } from 'enzyme';
import ReactDOM from 'react-dom';
import SideNav from './SideNav.js';

describe('Side Nav', () => {

  it('should match the snapshot', () => {

    const wrapper = shallow(<SideNav />);

    expect(wrapper).toMatchSnapshot();

  });

});
