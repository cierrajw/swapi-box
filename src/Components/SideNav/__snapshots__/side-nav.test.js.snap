// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Side Nav should match the snapshot 1`] = `
ShallowWrapper {
  Symbol(enzyme.__root__): [Circular],
  Symbol(enzyme.__unrendered__): <SideNav />,
  Symbol(enzyme.__renderer__): Object {
    "batchedUpdates": [Function],
    "getNode": [Function],
    "render": [Function],
    "simulateError": [Function],
    "simulateEvent": [Function],
    "unmount": [Function],
  },
  Symbol(enzyme.__node__): Object {
    "instance": null,
    "key": undefined,
    "nodeType": "host",
    "props": Object {
      "children": "The SideNav",
    },
    "ref": null,
    "rendered": "The SideNav",
    "type": "div",
  },
  Symbol(enzyme.__nodes__): Array [
    Object {
      "instance": null,
      "key": undefined,
      "nodeType": "host",
      "props": Object {
        "children": "The SideNav",
      },
      "ref": null,
      "rendered": "The SideNav",
      "type": "div",
    },
  ],
  Symbol(enzyme.__options__): Object {
    "adapter": ReactSixteenAdapter {
      "options": Object {
        "enableComponentDidUpdateOnSetState": true,
        "lifecycles": Object {
          "componentDidUpdate": Object {
            "onSetState": true,
          },
          "getDerivedStateFromProps": true,
          "getSnapshotBeforeUpdate": true,
          "setState": Object {
            "skipsComponentDidUpdateOnNullish": true,
          },
        },
      },
    },
  },
}
`;
